clearwater-infrastructure/usr/share/clearwater/infrastructure/scripts/1hosts

```diff
@@ -41,8 +41,9 @@
 # If the local IP address is an IPv6 address then the system needs a dummy
 # hostname that maps to the address.
 
-# Remove any previous dummy hostname from the hosts file.
-grep -v ' # maps to local IPv6 address' /etc/hosts > /tmp/hosts.$$
+# Remove any previous dummy hostname from the hosts file.  Also delete any
+# previous IPv6 localhost entries.
+grep -v ' # maps to local IPv6 address' /etc/hosts | grep -v '::1 localhost' > /tmp/hosts.$$
```

Comment:
I think it's better to grep out lines based on a comment. In particular, it looks like CentOS has a line `::1         localhost localhost.localdomain localhost6 localhost6.localdomain6` in /etc/hosts - if we ran this script on it I think we'd remove that line and re-add just `::1 localhost`, which would lose information. Maybe give all the lines in this script the comment `# added by clearwater-infrastructure 1hosts script`?